components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/src/main/java/org/wso2/carbon/apimgt/rest/api/endpoint/registry/util/EndpointRegistryMappingUtils.java

```diff
@@ -98,4 +100,18 @@ public static EndpointRegistryEntry fromDTOToRegistryEntry(RegistryEntryDTO regi
         registryEntry.setRegistryId(registryId);
         return registryEntry;
     }
+
+    /***
+     * Converts the sort by object according to the input
+     *
+     * @param sortBy
+     * @return Updated sort by field
+     */
+    public static String getRegistriesSortByField(String sortBy) {
```

Comment:
would this work for other sortBy fields except ID and regName?

Reply:
Currently, we have not given any other sorting fields in the REST API. Once we add them, we need to update this method